packages/search-ui/src/Results/components/Result/index.tsx

```diff
@@ -160,27 +160,68 @@ const Result = React.memo(
       );
     };
 
+    const renderPreviewImages = () => {
+      const getSetActive = useCallback(
+        (url: string, i: number) => () => {
+          setImageSrc(url);
+          setActiveImageIndex(i);
+        },
+        [],
+      );
+      return (
+        <Box
+          css={styles.previewImagesContainer}
+          aria-label={t('previewImagesContainer', { product: decodeHTML(title) })}
+        >
+          {image &&
+            isArray(image) &&
+            image.map((url, i) => {
```

Comment:
Shoud `isArray` be enough for the case?

```suggestion
          {isArray(image) &&
            image.map((url, i) => {
```